assets/src/hooks/useNearestIntersection.ts

```diff
@@ -4,15 +4,15 @@ import { FetchResult, loading, ok, fetchError } from "../util/fetchResult"
 import { useApiCall } from "./useApiCall"
 
 /**
- * @deprecated use {@linkcode useNearestIntersection2}
+ * @deprecated use {@linkcode useNearestIntersection}
  *
- * [Refactor `useNearestIntersection` away from `FetchResult`](https://app.asana.com/0/1148853526253420/1207213296687090/f)
+ * [⚙️ Refactor `useNearestIntersectionFetchResult` away from `FetchResult`](https://app.asana.com/0/1148853526253420/1207213296687090/f)
```

Comment:
I'm curious - did JJ or some kind of force-pushing bring this emoji back?

Reply:
I'm sorry I didn't catch this! I don't know how it got back in, probably just human error when resolving conflicts.
https://github.com/mbta/skate/pull/2581